src/config.rs

```diff
@@ -217,6 +224,23 @@ impl Default for BookConfig {
     }
 }
 
+/// Configuration for the build procedure.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(default, rename_all = "kebab-case")]
+pub struct BuildConfig {
```

Comment:
Now we've got a `BuildConfig`, what are your thoughts on removing `build_dir` from the `BookConfig` and putting it here? I think that'd make sense seeing as `build_dir` is more of a build option than metadata about your book.

Reply:
That seems logical.  I've implemented this as well as updating some of the documentation surrounding the build directory.